components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <code class="algorithms__code-cell">
+          <pre class="algorithms__code-cell__line">pip install qiskit</pre>
+          <pre class="algorithms__code-cell__line">pip install qiskit-ibm-provider</pre>
+        </code>
+        <!--
+          @click="[copyToClipboard($event), $trackClickEvent(`Copy ${title} install code`, 'providers')]"
+        -->
+        <cv-button
+          title="Copy"
+          size="sm"
+          class="algorithms__copy-button"
+          @click="copyToClipboard($event)"
+        >
+          Copy
+        </cv-button>
+      </div>
```

Comment:
You could replace this code with the `CodeSnippet` component [I suggested to create on this PR review](https://github.com/Qiskit/qiskit.org/pull/3000#discussion_r1129176985)

Reply:
Resolved in https://github.com/Qiskit/qiskit.org/pull/3020/commits/d60f48b9a31ffecbe3e17f577ebfa01892204e93